src/univariate/continuous/pgeneralizedgaussian.jl

```diff
@@ -0,0 +1,131 @@
+"""
+    PGeneralizedGaussian(α, μ, p)
+
+The *p-Generalized Gaussian distribution*, more commonly known as the exponential
+power or the generalized normal distribution, with scale 'α', location 'μ', and
```

Comment:
Do `'`-ticks have semantic/syntactic meaning?

Reply:
I believe so, I was using other distribution docstrings as a template for documenting the distribution. This can be seen in LocationScale.jl as well as Normal.jl